plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -46,11 +46,120 @@ States are the central elements of a state machine. A state has to be placed ins
 
 h3(#transition). Transition
 
-A transition is the transfer of one state to another. Transitions are diagrammed as arrows. They can have events and actions, but don't need to.
+A transition is the transfer of one state to another. A transition is diagrammed as a arrow leading from a _source state_ to a _target state_. A transition can also have
+* an _event trigger_, specifying which event(s) trigger the transition,
+* a _guard_, specifying which condition(s) must be fulfilled to trigger the transition,
+* an _effect_, specifying actions like assignments to variables, operations calls etc. that are to take place when this transition is executed.
 
-The syntax of events and actions is defined by a "textual description language":#the-statechart-language. Please refer to "section &quot;Events&quot;":#events for more details. "Section &quot;Reaction triggers&quot;":#reaction-triggers has more on actions.
+All of these three elements are optional, i.&nbsp;e. a transition may have one or more or them, but it doesn't need to. Events triggers, guard conditions, and effects are specified as a text that is attached to the transition arrow in the diagram.
 
-If a state has more than a single outgoing transition without event, then among those transitions the one that has been modeled first will be carried out.
+Here's an example:
+
+p(#fig_transition-example). 
+!(standard-image)images/docu_transition_030_example_01.png(Transition example)!
+
+p=. Transition example: A simple state machine to launch a rocket. The initial transition's _effect_ sets the counting variable _t_ to 10. As long as that variable's value is greater than 0, each second it gets counted down by 1. The operation _speak_ – executed as one of a transition's _actions_ – cares for audible information on how many seconds to go until rocket launch. If _t_ has been counted down to 0 – checked by a transition's _guard condition_ –, the rocket is launched. The countdown can be aborted anytime before launch by a _cancel_ event which _triggers_ the appropriate transition to fire.
+
+"Figure &quot;Transition syntax overview&quot;":#fig_transition-syntax-overview shows how the syntax of a transition specification is defined. You can refer to this diagram while reading the subsequent sections.
+
+p(#fig_transition-syntax-overview). 
+!(standard-image)images/docu_grammar_030_transition-overview.png(Transition syntax overview)!
+
+p=. Transition syntax overview
+
+h4(#unspecified-transition). Unspecified transition
+
+In the most simple case we have only the transition's arrow leading from a source state to a target state without any additional specification. In other words, there is no text attached to the transition. However, such an event will never be taken, so better make sure to set up a transition specification.
```

Comment:
That is not true. When nothing is specified, (1) we get a warning and (2) that transition is always taken.